src/CMakeLists.txt

```diff
@@ -220,7 +220,7 @@ if(PKG_CONFIG_FOUND)
   configure_valhalla_pc()
 
   install(FILES
-    ${CMAKE_BINARY_DIR}/libvalhalla.pc
+    ${VALHALLA_BUILD_DIR}/libvalhalla.pc
```

Comment:
huh, curious, for me that's the exact same statement. this failed somehow for you?